plugins/ingest/src/main/java/org/elasticsearch/plugin/ingest/transport/simulate/SimulatePipelineRequest.java

```diff
@@ -96,9 +97,13 @@ public void writeTo(StreamOutput out) throws IOException {
         static final String PIPELINE = "pipeline";
         static final String DOCS = "docs";
         static final String SOURCE = "_source";
-        static final String INDEX = "_index";
-        static final String TYPE = "_type";
-        static final String ID = "_id";
+        static final String INDEX = MetaData.INDEX.getFieldName();
+        static final String TYPE = MetaData.TYPE.getFieldName();
+        static final String ID = MetaData.ID.getFieldName();
+        static final String ROUTING = MetaData.ROUTING.getFieldName();
+        static final String PARENT = MetaData.PARENT.getFieldName();
+        static final String TIMESTAMP = MetaData.TIMESTAMP.getFieldName();
+        static final String TTL = MetaData.TTL.getFieldName();
```

Comment:
sorry I thought we were using XContentBuilderString here, does it make sense to remove these 7 contants and use MetaData ones directly everywhere?